pkg/chain/ethereum/ethereum.go

```diff
@@ -507,3 +509,17 @@ func (ec *EthereumChain) GetOpenedTimestamp(keepAddress common.Address) (time.Ti
 
 	return keepOpenTime, nil
 }
+
+// WeiBalanceOf returns the given address balance in WEI.
```

Comment:
Maybe we should add to this comment that it is a balance of the latest known block? Something like:
```
WeiBalanceOf returns the wei balance of the given address from the latest known block.
```